pkg/component/runtime/manager_test.go

```diff
@@ -2331,13 +2547,14 @@ func TestManager_FakeShipper(t *testing.T) {
 		t.Fatalf("failed early: %s", err)
 	}
 
-	endTimer := time.NewTimer(2 * time.Minute)
+	timeout := 3 * time.Minute
```

Comment:
Nitpicking here: is the test a minute slower now?

Reply:
Good catch.  I think I made this change for debugging and it can be reverted to 2 minutes.  Will change and try the test again.  Thanks.

Reply:
Fixed.